### DISCUSSION TITLE
How to prevent or block editing?

### DISCUSSION THREAD

**Discussion by sibstark** [bangle-io/bangle.dev#250] [General]
Hello! I would like to ask if there are any ways to prevent or block editing area?
I want to allow to edit only after user pushes the "Edit" button.

![image](https://user-images.githubusercontent.com/15224416/160758226-b7755d7c-25af-42eb-82dc-72c9c4488274.png)

**Comment by kepta** [comment]
You can create a new Prosemirror plugin and use [editable](https://prosemirror.net/docs/ref/#view.EditorView.editable) prop to enable or disable the editing of your editor:
```js
function editingAllowedPlugin() {
    return new Plugin({
      props: {
        editable(state) {
      
          return isEditingAllowed(state) ; // your logic for enabling or disabling 
        },
      },
    });
```

**Reply by sibstark** [reply]
Thank you!
